### DISCUSSION TITLE
Keystone admin custom browser title and favicon

### DISCUSSION THREAD

**Discussion by molotow11** [keystonejs/keystone#8407] [Questions] [answered]
Hi, Guys!
Is it possible to change admin side pages <title> and favicon?

**Comment by arsekil** [comment, marked as answer]
https://github.com/keystonejs/keystone/discussions/7506#discussion-4042133

**Reply by molotow11** [reply]
Thanks @andrejmoltok

**Reply by jj-matos** [reply]
Same here. Tried everything:

- Changed the icon file directly through ftp
- Different browsers
- Clearing cache and incognito mode
- Flush DNS
- Checked if there was any cache on server level. There is none.
- The Keystone icon is nowhere in the files

There are times where even after login, the icon seems to change between the old and the new. Like flickering.

Is it possible that the app is storing the old icon in some CDN. And if yes, how to flush that?
